docs/src/content/docs/es/utilities/Injectors/create-injection-token.md

```diff
@@ -0,0 +1,240 @@
+---
+title: createInjectionToken
+description: Crea un InjectionToken y devuelve un injectFn y provideFn para el mismo.
+badge: stable
+contributor: chau-tran
+---
+
+`createInjectionToken` es una abstacción sobre la creación de un [`InjectionToken`](https://angular.io/api/core/InjectionToken) y devuelve una tupla de `[injectFn, provideFn, TOKEN]`
+
+Crear un `InjectionToken` no suele ser un gran problema, pero consumir el `InjectionToken` puede ser un poco tedioso si el proyecto utiliza `InjectionToken` mucho.
+
+```ts
+import { createInjectionToken } from 'ngxtension/create-injection-token';
+```
+
+## Uso
+
+```ts
+function countFactory() {
+	return signal(0);
+}
+
+export const [
+	injectCount,
+	/* provideCount */
+	/* COUNT */
+] = createInjectionToken(countFactory);
+
+@Component({})
+export class Counter {
+	count = injectCount(); // WritableSignal<number>
+	/* count = inject(COUNT); // WritableSignal<number> */
+}
+```
+
+### `CreateInjectionTokenOptions`
+
+`createInjectionToken` acepta un segundo argumento de tipo `CreateInjectionTokenOptions` que permite personalizar el `InjectionToken` que estamos creando:
+
+```ts
+export interface CreateInjectionTokenOptions<T = unknown> {
+	isRoot?: boolean;
+	deps?: unknown[];
+	extraProviders?: Provider[];
+	multi?: boolean;
+	token?: InjectionToken<T>;
+}
+```
+
+#### `isRoot`
+
+Por defecto, `createInjectionToken` crea un token `providedIn: 'root'` por lo que no tenemos que _proveerlo_ para usarlo. Para crear un token no-root, pasa `isRoot: false`
+
+```ts
+export const [injectCount, provideCount] = createInjectionToken(countFactory, {
+	isRoot: false,
+});
+
+@Component({
+	providers: [provideCount()],
+})
+export class Counter {
+	count = injectCount(); // WritableSignal<number>
+	/* count = inject(COUNT); // WritableSignal<number> */
+}
+```
+
+#### `deps`
+
+En muchos casos, `InjectionToken` puede depender de otros `InjectionToken`. Aquí es donde entran en juego `deps` y lo que podemos pasar en `deps` depende de la firma de `factoryFn` que acepta `createInjectionToken`.
+
+```ts
+export const [, , DEFAULT] = createInjectionToken(() => 5);
+
+function countFactory(defaultValue: number) {
+	return signal(defaultValue);
+}
+
+export const [injectCount, provideCount] = createInjectionToken(countFactory, {
+	isRoot: false,
+	deps: [DEFAULT],
+});
+```
+
+#### `extraProviders`
+
+We can also pass in other providers, via `extraProviders`, to `createInjectionToken` so when we call `provideFn()`, we provide those providers as well.
```

Comment:
Looks like this one is translated up until this line, it is also a large one, is this intentional?